### PR TITLE
feat(pokedex):Reconfig sizing units for the pokedex app

### LINKED ISSUE #19: Resize pokedex model
**Is your feature request related to a problem? Please describe.**
Pokedex model is too small

**Describe the solution you'd like**
Resize the pokedex model at the start of the app
ere.

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Change how the pokedex Model resizes

## Purpose
On different screens the pokedex is either too small or too big

## Approach
changed size untis to change based on the screen it is in with percentages 

Closes #19

### CODE REVIEW

File: pokemon-app/src/App.css [hunk 1]
```diff
@@ -1,67 +1,57 @@
...
+  animation-timing-function: ease-in-out;
+  animation-name: increase;
+  margin-left: 20vw;
 }
-#dex-right-open-opening{
-  width: 0px;
+@keyframes increase{
+  from{height: 0vh; width: 0vw;}
+  to{height: 130vh; width: 64em;}
+}
```
Comment (BlackCubes) [inline pokemon-app/src/App.css:12]: Lots of keyframes, but it looks gewd!

File: pokemon-app/src/App.css [hunk 2]
```diff
@@ -71,20 +61,18 @@
...
+  animation-delay: 2s;
   animation-fill-mode: forwards;
 }
 .splash-title{
   position: absolute;
   height: 0%;
-  width: 0%;
   animation: pop 1s;
-  animation-delay: 1.5s;
+  animation-delay: 2.4s;
```
Comment (BlackCubes) [inline pokemon-app/src/App.css:72]: Dat fine details 😲

File: pokemon-app/src/App.css [hunk 3]
```diff
@@ -1,67 +1,57 @@
...
-  animation-delay: 1s;
+  top: 8.2%;
+  left: 94%;
+  height: 91.8%;
+  width: 0%;
+  animation-name: open-right;
+  animation-delay: 2.3s;
   animation-duration: 1s;
   animation-fill-mode: forwards;
   animation-timing-function: ease-out;
```
Comment (Wolftownn) [inline pokemon-app/src/App.css:41]: Awesome creativity with animations!

**Review by Wolftownn — APPROVED** [review]
Looks great!

**Review by BlackCubes — APPROVED** [review]
Beautiful. No words can't describe the simplicity

**Pull request merged**